examples/cli/src/main/java/io/unitycatalog/cli/utils/CliUtils.java

```diff
@@ -31,11 +31,13 @@ public class CliUtils {
 
   public static final String FUNCTION = "function";
   public static final String REGISTERED_MODEL = "registered_model";
+  public static final String MODEL_VERSION = "model_version";
```

Comment:
So I talked with TD about this.  I think we want to leave the model version param as we have it because there are slight differences between the flags that need to be set/specified for models and model versions.

For example, on create, you only specify the run_id and source on a model version.  On a list, you only specify catalog and schema for registered models, but must include the model name for model versions.

There are enough differences that it probably would make the CLI slightly confusing to use if you had to reason about different required flags based on the presence of a --version flag.

Reply:
sure ... let's keep it completely separate.